messaging.rst

```diff
@@ -85,27 +85,80 @@ Messages
+- The lock's amount must be smaller than the current :term:`capacity`. If the amount is higher then the recipient will reject it, as it means he will be spending money it does not own. .
 - The lock expiration must be greater than the current block number.
 - The :term:`transferred amount` must not change.
 
-Fields
-^^^^^^
+Matrix message
+^^^^^^^^^^^^^^
+
+On the transport level ``LockedTransfer`` is encoded as a JSON message. The message then might hop through a different
+number of nodes until it reaches the ``target``. 
+
+Let's assume that there is a network:
+
+- [A] `0x540B51eDc5900B8012091cc7c83caf2cb243aa86`  
+- [B] `0x2A915FDA69746F515b46C520eD511401d5CCD5e2`
+- [C] `0x811957b07304d335B271feeBF46754696694b09e`
+
+Where **A** has a channel with **B** and **B** has a channel with **C**.
+
+A <---> B <---> C
+
+If **A** wants to send 10 wei of a Token(0xc778417e063141139fce010982780140aa0cd5ab) to **C** he has to first
+send a LockedTransfer with **B** (``recipient``) where **C** is the  ``target``. After receiving the message, **B** has to send a new LockedTransfer to **C**.
+
+The message that will be sent from A -> B over the matrix transport would look like this.
+
+.. code-block:: json
+
+    {
+        "type": "LockedTransfer",
+        "chain_id": 337,
+        "message_identifier": 123456,
+        "payment_identifier": 1,
+        "nonce": 1,
+        "token_network_address": "0xe82ae5475589b828D3644e1B56546F93cD27d1a4",
+        "token": "0xc778417E063141139Fce010982780140Aa0cD5Ab",
+        "channel_identifier": 1338,
+        "transferred_amount": 0,
+        "locked_amount": 10,
+        "recipient": "0x2A915FDA69746F515b46C520eD511401d5CCD5e2",
+        "locksroot": "0x607e890c54e5ba67cd483bedae3ba9da9bf2ef2fbf237b9fb39a723b2296077b",
+        "lock": {
+            "type": "Lock",
+            "amount": 10,
+            "expiration": 1,
+            "secrethash": "0x59cad5948673622c1d64e2322488bf01619f7ff45789741b15a9f782ce9290a8"
+        },
+        "target": "0x811957b07304d335B271feeBF46754696694b09e",
+        "initiator": "0x540B51eDc5900B8012091cc7c83caf2cb243aa86",
+        "fee": 0,
+        "signature": "0x33b336f151f9790f40287655bd412a043be83a03d0136ef5e002229dd04d5b4c2b505b65911251b2a2eb428403de394064bdae0cd8d4a3bb47a10b1a0d924b921c"
+    }
+
+Message Fields
+^^^^^^^^^^^^^^
 
 This should correspond to `the packed format of LockedTransfer <https://github.com/raiden-network/raiden/blob/d504ed25b85eea5738fd3d2149bd8392a2b02226/raiden/encoding/messages.py#L164>`_.
 
+Let's call this structure of message fields ``message_structure`` from now on. Also let's assume that there is 
```

Comment:
why didn't you call that message structure in the first place? Or is it the message structure that consists of various message fields?